sequencer/node/node.go

```diff
@@ -653,27 +645,25 @@ func (n *Node) StartSynchronizer() {
 						log.Errorw("Synchronizer.Sync", "err", err)
 					}
 				}
+			case msg := <-n.msgCh:
+				if err := n.coord.HandleMsg(n.ctx, msg);  n.ctx.Err() != nil {  // We will use forger here to handle the message
```

Comment:
It will create multiple if loops here, currently I dont think readability is an issue here. This line is only checking for ctx error and later it checks for function error.